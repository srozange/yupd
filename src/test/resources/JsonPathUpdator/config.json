{
  "server": {
    "port": 8080,
    "host": "localhost",
    "ssl": false
  },
  "database": {
    "url": "jdbc:postgresql://localhost:5432/mydb",
    "username": "user",
    "maxConnections": 10,
    "maxTimeout" : 20
  },
  "features": {
    "enableCaching": false,
    "enableMetrics": true
  }
}