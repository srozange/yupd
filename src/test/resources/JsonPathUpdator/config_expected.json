{
  "server" : {
    "port" : 9090,
    "host" : "localhost",
    "ssl" : false
  },
  "database" : {
    "url" : "jdbc:postgresql://localhost:5432/mydb",
    "username" : "user",
    "maxConnections" : 10
  },
  "features" : {
    "enableCaching" : true,
    "enableMetrics" : false
  }
}